common/taskana-common-data/src/main/resources/sql/sample-data/task.sql

```diff
@@ -1,101 +1,102 @@
 -- sample-data is used for rest tests and for the example application
 
--- TASK TABLE          (ID                                        , EXTERNAL_ID                               , CREATED             , CLAIMED              , COMPLETED            , modified             , received             , planned              , due                  , name                      , creator                      , description                                , note                                     , priority, manual_priority, state       , classification_category  , classification_key, classification_id                         , workbasket_id                              , workbasket_key, domain    , business_process_id, parent_business_process_id, owner         , por_company , por_system , por_system_instance, por_type , por_value    , is_read, is_transferred, callback_info, callback_state, custom_attributes , custom1     , custom2     , custom3,   , custom4    , custom5    , custom6    , custom7    ,custom8     ,custom9     ,custom10    ,custom11,    ,custom12     ,custom13     ,custom14  ,custom15     ,custom16    , custom-int-1, custom-int-2, custom-int-3, custom-int-4, custom-int-5, custom-int-6, custom-int-7, custom-int-8
```

Comment:
What did you modify in this file?

Reply:
I added a new Task (TKI:000000000000000000000000000000000104) with the state READY_FOR_REVIEW:

![image](https://user-images.githubusercontent.com/60552466/196626503-dd57bb09-7bf6-4bd6-9d7d-3cf1dd675c2f.png)